integration-tests/utils/integration-utils.js

```diff
@@ -107,6 +107,17 @@ const findInDataTableRow = (page, entityName, text) => {
   return findElement(page, `//*[@role="grid"]//*[contains(.,"${entityName}")]/following-sibling::*[contains(.,"${text}")]`)
 }
 
+const openNotification = async page => {
+  //close out any non-error notifications first
+  await dismissNotifications(page)
+
+  const errorDetails = await page.$x('(//a | //*[@role="button"] | //button)[contains(normalize-space(.),"Details")]')
```

Comment:
```suggestion
  const [errorDetails] = await page.$x('(//a | //*[@role="button"] | //button)[contains(normalize-space(.),"Details")]')
```

Reply:
I imagine this way is essentially saving the result of the await page.$x... function as an array. However how would this handle if that result was empty? Would it save errorDetails to be an empty array? If so isn't an empty array truthy?

Reply:
`emptyArray[0]` is `undefined`

Reply:
We're destructuring the array here, `errorDetails` is `array[0]`

Reply:
The `await page.$x...` function returns an empty array already when it doesn't find any elements that match. I tried using the change you suggested but then it returns `undefined` instead which doesn't seem preferable to me

Reply:
Sure, if you prefer; we're doing `errorDetails[0]` below, so that's the same undefined. Not a huge difference, feel free to take it or leave it.